docs/en/02_Developer_Guides/01_Templates/11_Partial_Template_Caching.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Partial Template Caching
+summary: Cache a section of a template Reduce rendering time with cached templates and understand the limitations of the ViewableData object caching.
+icon: tags
+---
+
+## Partial template caching
+
+Partial template caching is a feature that allows caching of rendered portions of templates. Cached content
+is fetched from a [cache backend](../performance/caching), instead of being regenerated repeatedly.
+
+
+### Base syntax
+
+Here is an example of the basic usage. This is not a definitive example of the syntax, but it shows the most
+common use case.
+
+[note]
+See also [Complete Syntax definition](#complete-syntax-defintition) section
+[/note]
+
+```ss
+<% cached $CacheKey if $Condition %>
+  <!-- Content block -->
+<% end_cached %>
+```
+
+The key parts are `$CacheKey`, `$Condition` and `<!-- Content block -->`.  
```

Comment:
Content block may be confusing with elemental content blocks.  Should probably just say 'Content' (though that may be confused with SiteTree.Content) or 'Cached HTML'

Reply:
Good catch. I guess we should change it to `<!-- Content -->`?